app/assets/stylesheets/invitations.css.scss

```diff
@@ -5,11 +5,19 @@
 }
```

Comment:
I'm actually against using relative font-sizes as percentages, ems, or rems. Is a nice feature but in my experience it was only useful when having to resize something dinamycally as old newspaper sites did with the text body. Ems/rems also make hard to tell what a font size will look like when reading/writing code.

Generally, I prefer using specific pixel size for the fonts, and change it (if needed) in other form factors/viewport sizes by media queries.